lib/std/time/epoch.zig

```diff
@@ -40,3 +43,179 @@ pub const morphos = amiga;
 pub const brew = gps;
 pub const atsc = gps;
 pub const go = clr;
+
+/// The type that holds the current year, i.e. 2016
+pub const Year = u16;
+
+pub const epoch_year = 1970;
+pub const secs_per_day: u17 = 24 * 60 * 60;
+
+pub fn isLeapYear(year: Year) bool {
+    if (math.comptimeMod(year, 4) != 0)
+        return false;
+    if (math.comptimeMod(year, 100) != 0)
+        return true;
+    return (0 == math.comptimeMod(year, 400));
+}
+
+test "isLeapYear" {
+    try testing.expectEqual(false, isLeapYear(2095));
+    try testing.expectEqual(true, isLeapYear(2096));
+    try testing.expectEqual(false, isLeapYear(2100));
+    try testing.expectEqual(true, isLeapYear(2400));
+}
+
+pub fn getDaysInYear(year: Year) u9 {
+    return if (isLeapYear(year)) 366 else 365;
+}
+
+pub const YearLeapKind = enum(u1) { not_leap, leap };
+
+pub const Month = enum(u4) {
+    jan,
```

Comment:
```suggestion
    jan = 1,
```

And so forth. You can avoid the +1 dance and the `numeric` helper.

Reply:
sure that should work, done